starknet_py/net/models/transaction.py

```diff
@@ -105,6 +120,20 @@ def calculate_hash(self, chain_id: StarknetChainId) -> int:
             nonce=self.nonce,
         )
 
+    @staticmethod
+    def _contract_attribute_name(data: Dict[str, Any]):
+        return (
+            "contract_definition" if "contract_definition" in data else "contract_class"
+        )
+
+    @staticmethod
+    def _compress_program(program: Dict[str, Any]):
+        compressed_program = json.dumps(program)
+        compressed_program = gzip.compress(data=compressed_program.encode("ascii"))
+        compressed_program = base64.b64encode(compressed_program)
+
+        return compressed_program.decode("ascii")
```

Comment:
Do we really need these two as static methods?

Reply:
Ok, I see what is the problem. Let's keep it as it is

Reply:
Compress program could be inlined. Attribute name would be duplicated in 2 places.